x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/resilient.ts

```diff
@@ -241,52 +234,53 @@ export default function resilientTest({ getService }: FtrProviderContext) {
+              expect(resp.body.connector_id).to.eql(resilientActionId);
               expect(resp.body.status).to.eql('error');
             });
         });
 
         it('should handle failing with a simulated success without unsupported action', async () => {
           await supertest
-            .post(`/api/actions/connector/${simulatedActionId}/_execute`)
+            .post(`/api/actions/connector/${resilientActionId}/_execute`)
             .set('kbn-xsrf', 'foo')
             .send({
               params: { subAction: 'non-supported' },
             })
             .then((resp: any) => {
               expect(resp.body).to.eql({
-                connector_id: simulatedActionId,
+                connector_id: resilientActionId,
                 status: 'error',
-                retry: false,
-                message:
-                  'error validating action params: types that failed validation:\n- [0.subAction]: expected value to equal [getFields]\n- [1.subAction]: expected value to equal [getIncident]\n- [2.subAction]: expected value to equal [handshake]\n- [3.subAction]: expected value to equal [pushToService]\n- [4.subAction]: expected value to equal [incidentTypes]\n- [5.subAction]: expected value to equal [severity]',
+                retry: true,
+                message: 'an error occurred while running the action',
+                service_message: `Sub action "non-supported" is not registered. Connector id: ${resilientActionId}. Connector name: IBM Resilient. Connector type: .resilient`,
                 errorSource: TaskErrorSource.FRAMEWORK,
               });
             });
         });
 
         it('should handle failing with a simulated success without subActionParams', async () => {
           await supertest
-            .post(`/api/actions/connector/${simulatedActionId}/_execute`)
+            .post(`/api/actions/connector/${resilientActionId}/_execute`)
             .set('kbn-xsrf', 'foo')
             .send({
               params: { subAction: 'pushToService' },
             })
             .then((resp: any) => {
               expect(resp.body).to.eql({
-                connector_id: simulatedActionId,
+                connector_id: resilientActionId,
                 status: 'error',
-                retry: false,
-                message:
-                  'error validating action params: types that failed validation:\n- [0.subAction]: expected value to equal [getFields]\n- [1.subAction]: expected value to equal [getIncident]\n- [2.subAction]: expected value to equal [handshake]\n- [3.subActionParams.incident.name]: expected value of type [string] but got [undefined]\n- [4.subAction]: expected value to equal [incidentTypes]\n- [5.subAction]: expected value to equal [severity]',
+                retry: true,
+                message: 'an error occurred while running the action',
+                service_message:
+                  "Cannot destructure property 'externalId' of 'incident' as it is undefined.",
```

Comment:
What is the reason that the schema validation is not happening any more?

Reply:
Same. PushToService schema of CaseConnector doesn't validate it.